v-next/hardhat-node-test-reporter/src/formatting.ts

```diff
@@ -130,6 +130,7 @@ export function formatUnusedDiagnostics(
       if (isTestOnlyMessage(message)) {
         return testOnlyMessage ?? message;
       }
+      return message;
```

Comment:
This is why we weren't displaying the error details at all before.

Reply:
Ouch 😓

Great catch